Observatory-Control-System/Safety-Systems/Control-Safety-Systems.rst

```diff
@@ -94,7 +94,7 @@ It is intended for use in an EMERGENCY. It is not a safeguard, but is considered
 
```

Comment:
**Not able to add comment on line 29:"Linked here" seems to not follow the writing style guidelines.
**Not able to add comment on line 71,72: I think these lines need rewriting.
**Not able to add comment on line 83:** AuxTel second floor.

Reply:
I didn't edit this file at all since it was written, but we now have an updated version for GIS in the Confluence ready to be in our repo. Should be up soon.